main.go

```diff
@@ -38,6 +38,8 @@ func main() {
 		updated          = flag.Bool("c", true, `Print changed docs`)
 		changed          = flag.Bool("a", true, `Print added docs`)
 		deleted          = flag.Bool("d", true, `Print deleted docs`)
+		//replace
+		replaceIdAnother = flag.String("replaceIdAnother", "", `replace id field to other field you want`)
```

Comment:
I'd prefer to use a simple `replace` or `replace-with`. Please use all lowercase and dashes.

Reply:
Thank you, I will correct it and submit it again, thanks to the author